### PR TITLE
[8.0] mrp_analytic

### PR DESCRIPTION
**Pull request by pedrobaeza**
# 
# Analytic for manufacturing

This module links the manufacturing area with analytic management.

For now, it adds analytic account configuration for production orders,
and their corresponding shortcuts. Other modules can take advantage of
this link for other usages.
# Usage

There is a new field named "Analytic Account" in the Production Order.

You can also navigate from the Analytic Account to the related Production
Orders.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@jbeficent, I have started from your work to finish the module.

@rafaelbn, ready to review.

**Comment by pedrobaeza** [comment]
We need a fast review+merge because there are several modules depending on this one in other repos (manufacture).

File: mrp_analytic/README.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Analytic for manufacturing
+==========================
+
+This module links the manufacturing area with analytic management.
+
+For now, it adds analytic account configuration for production orders,
```
Comment (elicoidal) [inline mrp_analytic/README.rst]: s/production/manufacturing

File: mrp_analytic/README.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+and their corresponding shortcuts. Other modules can take advantage of
+this link for other usages.
+
+Usage
+=====
+
+There is a new field named "Analytic Account" in the Production Order.
```
Comment (elicoidal) [inline mrp_analytic/README.rst]: idem

File: mrp_analytic/README.rst [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+You can also navigate from the Analytic Account to the related Production
```
Comment (elicoidal) [inline mrp_analytic/README.rst]: idem

**Comment by pedrobaeza** [comment]
@elicoidal, done!

**Comment by rafaelbn** [comment]
Tested in runbot :+1:

**Comment by JavierIniesta** [comment]
Tested in runbot :+1:

**Comment by JMODaERP** [comment]
I tested in runbot: OK: :+1: 

2015-12-03 13_35_25-manufacturing orders - odoo

2015-12-03 13_35_12-odoo

**Comment by JordiBForgeFlow** [comment]
Tested in runbot: :+1:

**Comment by pedrobaeza** [comment]
As everything is honoured and have enough reviews, I merge.

**Pull request merged**